django_modern_rest/endpoint.py

```diff
@@ -525,10 +552,17 @@ def modify(  # noqa: WPS211
     *,
     status_code: HTTPStatus | Empty = EmptyObj,
     headers: Mapping[str, NewHeader] | Empty = EmptyObj,
-    validate_responses: bool | Empty = EmptyObj,
+    validate_responses: bool | None = None,
     extra_responses: list[ResponseDescription] | Empty = EmptyObj,
-    error_handler: SyncErrorHandlerT | AsyncErrorHandlerT | Empty = EmptyObj,
+    error_handler: SyncErrorHandlerT | AsyncErrorHandlerT | None = None,
     allow_custom_http_methods: bool = False,
+    summary: str | None = None,
+    description: str | None = None,
+    tags: list[str] | None = None,
+    operation_id: str | None = None,
+    deprecated: bool = False,
+    security: list[SecurityRequirement] | None = None,
+    external_docs: ExternalDocumentation | None = None,
```

Comment:
You need to update all `modify` overloads with this, these fields are always allowed. To catch this, please provide one optionl field like `description` in `typesafety/`